__mocks__/axios.js

```diff
@@ -0,0 +1,5 @@
+const Axios = {
```

Comment:
mocks required to Jest docs: https://jestjs.io/docs/en/manual-mocks#mocking-node-modules